content/integrations/faq/using-rbac-permission-with-your-kubernetes-integration.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Using RBAC permission with your Kubernetes integration
+kind: faq
+customnav: integrationsnav
+---
+
+Use these Kubernetes RBAC entities for your Datadog agent to properly configure the previous permissions by [applying this datadog service account to your pods](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/):
```

Comment:
What previous permissions? This sentence is confusing.

Reply:
👍